bundles/binding/org.openhab.binding.modbus/src/main/java/net/wimpi/modbus/net/TCPMasterConnection.java

```diff
@@ -223,4 +232,12 @@ public String toString() {
         return new ToStringBuilder(this, toStringStyle).append("socket", m_Socket).toString();
     }
 
+    public int getConnectTimeoutMillis() {
+        return m_ConnectTimeoutMillis;
+    }
+
+    public void setConnectTimeoutMillis(int m_ConnectTimeoutMillis) {
```

Comment:
I agree that it does not really make sense but I'm keeping the "jamod" (net.wimpi.modbus) code base consistent. 

For whatever reason they (=original authors of jamod) are prefixing the private variables with `m_`.

Perhaps at some point we should consider formatting the whole net.wimpi.modbus code to comform to modern java style guide.